Sources/DittoAllToolsMenu/Views/Tool Container Views/LoggingDetailsViewer.swift

```diff
@@ -0,0 +1,28 @@
+///
+//  LoggingDetailsViewer.swift
+//  DittoToolsApp
+//
+//  Created by Eric Turner on 6/1/23.
+//
+//  Copyright © 2023 DittoLive Incorporated. All rights reserved.
+
+import DittoExportLogs
+import DittoSwift
+import SwiftUI
+
+struct LoggingDetailsViewer: View {
+    
+    // @ObservedObject var dittoManager = DittoManager.shared
+    
+    var body: some View {
+        //LoggingDetailsView(loggingOption: $dittoManager.loggingOption)
+        Text("hello")
+#warning("TODO: fix Logging Details View")
+    }
+}
+
+//struct LoggingDetailsViewer_Previews: PreviewProvider {
+//    static var previews: some View {
+//        LoggingDetailsViewer(dittoManager: DittoManager.shared)
+//    }
+//}
```

Comment:
Is this whole view just dead code? Or is this used somewhere and we should log something to clean it up?